spec/controllers/teams_controller_spec.rb

```diff
@@ -12,11 +12,11 @@
 
   let(:valid_attributes) { build(:team).attributes.merge(roles_attributes: [{ name: 'coach', github_handle: 'tobias' }]) }
 
-  before do
-    user.roles.create(name: 'student', team: team)
-  end
-
   describe "GET index" do
```

Comment:
Now that the new ability rule prevents students from creating a second team, the specs can't assign two teams either. Move the one big `before` into the contexts.